yaksh/templates/yaksh/post_comments.html

```diff
@@ -0,0 +1,72 @@
+{% extends "user.html" %}
```

Comment:
This will show the student navbar even if I open the comments from the moderator dashboard course details page. You can pass the template name from views by checking whether the user is a moderator or not.